code/modules/clothing/head/misc_special.dm

```diff
@@ -175,6 +175,12 @@
 		else
 			set_light(0)
 
+/obj/item/clothing/head/pumpkinhead/attackby(var/obj/item/I, var/mob/user)
```

Comment:
I don't know why people might want to put accessories on a pumpkin head but you should probably call `..()` anyway